kolibri/plugins/learn/assets/src/composables/useDownloadRequests.js

```diff
@@ -140,15 +96,23 @@ export default function useDownloadRequests(store) {
   }
 
   function removeDownloadRequest(content) {
-    console.log(`requested removal of ${content.id}`);
-    Vue.delete(downloadRequestMap.downloads, content.id);
+    ContentRequestResource.deleteModel({
+      id: content.id,
+      contentnode_id: content.contentnode_id,
+    })
+      .then(Vue.delete(downloadRequestMap, content.id))
+      .then(set(downloadRequestMap, 'totalDownloads', Object.keys(downloadRequestMap).length));
     return Promise.resolve();
   }
 
   function removeDownloadsRequest(contentList) {
-    console.log(`requested removal of ${contentList.length} items`);
     contentList.forEach(content => {
-      Vue.delete(downloadRequestMap.downloads, content.id);
+      ContentRequestResource.deleteModel({
+        id: content.id,
+        contentnode_id: content.contentnode_id,
+      })
+        .then(Vue.delete(downloadRequestMap, content.id))
+        .then(set(downloadRequestMap, 'totalDownloads', Object.keys(downloadRequestMap).length));
```

Comment:
This is still setting `totalDownloads` on the `downloadRequestMap` and not storing it as an overall reactive property.